pkg/routeagent/main.go

```diff
@@ -61,14 +64,14 @@ func main() {
 		klog.Fatalf("Error building submariner clientset: %s", err.Error())
 	}
 
-	submarinerInformerFactory := submarinerInformers.NewSharedInformerFactoryWithOptions(submarinerClient, time.Second*30, submarinerInformers.WithNamespace(srcs.Namespace))
+	submarinerInformerFactory := submarinerInformers.NewSharedInformerFactoryWithOptions(submarinerClient, resyncInterval, submarinerInformers.WithNamespace(srcs.Namespace))
```

Comment:
@tpantelis it seems like this was the breaking change, may be setting to 0 does not disable the resync interval?

Reply:
It does but even if it didn't we shouldn't be seeing failures as resync has been occurring all along. It seems we're actually indirectly relying on the resync, ie something fails or doesn't work properly the first time around such that a subsequent resync fixes it. If so, I also wonder if this contributes to longer startup or fail-overs.